bazel/repositories.bzl

```diff
@@ -1,7 +1,7 @@
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
-ENVOY_COMMIT = "90fb3c8baa81651e0af865c298b1934df818a4dd"  # Apr 11, 2022
-ENVOY_SHA = "ed86a0f9e62fdb415cc7587419796a2afb99b0245592867c6cae2f55c84872f2"
+ENVOY_COMMIT = "03efacf2f9bc10bf92c61491274c4ba9ff0ad1bd"  # Apr 11, 2022
```

Comment:
Can we also update the date to Apr 13?

Alternatively consider removing the date comment, it is redundant to the revision history and is error prone.

Could you also merge main so that we trigger the new AZP pipeline on this PR? The pipeline was added in #827.